go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/mike1808/h264decoder
```

Comment:
I think this change wasn't intentional. Could you please revert the module name?

Reply:
You're right, I forgot change it after testing. Fixed now.